presto-main/src/main/java/com/facebook/presto/cost/RuntimePlanNodeStats.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cost;
+
+import com.facebook.presto.spi.statistics.RuntimeSourceInfo;
+import org.pcollections.HashTreePMap;
+
+import java.util.Arrays;
+
+// Represents stats collected during execution of the query.
+public class RuntimePlanNodeStats
+        extends PlanNodeStatsEstimate
+{
+    private final long[] bytesByPartitionId;
+
+    public RuntimePlanNodeStats(long[] bytesByPartitionId, double totalSize)
+    {
+        // We don't have info on row count. Setting it to MIN_VALUE till we figure it out
```

Comment:
set it to NAN instead to represent unknown

Reply:
PlanNodeStatsEstimate constructor has check for using NaN or Negative number. Setting it to NaN will require removing the check. Do we want to do this ?

Reply:
the check is that it has to be either NaN or positive.  NaN is used to represent an unknown value.

Reply:
missed that. Thanks for pointing it out